modules/videoio/src/cap_obsensor/obsensor_uvc_stream_channel.cpp

```diff
@@ -179,7 +179,7 @@ DepthFrameProcessor::DepthFrameProcessor(const OBExtensionParam& param) : param_
     }
 }
 
-DepthFrameProcessor::~DepthFrameProcessor()
```

Comment:
It is better to drop `noexcept` from the header file instead.

Reply:
Done.